openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/TableSorting.spec.ts

```diff
@@ -0,0 +1,444 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { APIRequestContext } from '@playwright/test';
+import { ApiEndpointClass } from '../../support/entity/ApiEndpointClass';
+import { DashboardDataModelClass } from '../../support/entity/DashboardDataModelClass';
+import { DatabaseSchemaClass } from '../../support/entity/DatabaseSchemaClass';
+import { EntityTypeEndpoint } from '../../support/entity/Entity.interface';
+import { FileClass } from '../../support/entity/FileClass';
+import { DatabaseServiceClass } from '../../support/entity/service/DatabaseServiceClass';
+import { PipelineServiceClass } from '../../support/entity/service/PipelineServiceClass';
+import { SpreadsheetClass } from '../../support/entity/SpreadsheetClass';
+import { StoredProcedureClass } from '../../support/entity/StoredProcedureClass';
+import { TableClass } from '../../support/entity/TableClass';
+import { TopicClass } from '../../support/entity/TopicClass';
+import { performAdminLogin } from '../../utils/admin';
+import { redirectToHomePage, testTableSorting } from '../../utils/common';
+import { test } from '../fixtures/pages';
+
+let apiContext: APIRequestContext;
+let afterAction: () => Promise<void>;
+
+// Shared services for "Services page" tests
+const databaseService1 = new DatabaseServiceClass();
+const databaseService2 = new DatabaseServiceClass();
+const pipelineService1 = new PipelineServiceClass();
+const pipelineService2 = new PipelineServiceClass();
+
+test.beforeAll(async ({ browser }) => {
+  test.slow();
+
+  const result = await performAdminLogin(browser);
+  apiContext = result.apiContext;
+  afterAction = result.afterAction;
+
+  // Create only the services needed for the "Services page" tests
+  await databaseService1.create(apiContext);
+  await databaseService2.create(apiContext);
+  await pipelineService1.create(apiContext);
+  await pipelineService2.create(apiContext);
+});
+
+test.afterAll(async () => {
+  test.slow();
+
+  await databaseService1.delete(apiContext);
+  await databaseService2.delete(apiContext);
+  await pipelineService1.delete(apiContext);
+  await pipelineService2.delete(apiContext);
+
+  await afterAction();
```

Comment:
Same here, please use the assets created in the common setup. Also, we don't need to delete the entities if we are creating them for tests.